code/ATMOSPHERICS/components/unary/vent_scrubber.dm

```diff
@@ -239,29 +218,29 @@
 		scrubbing = !scrubbing
 
 	if(signal.data["co2_scrub"] != null)
-		scrub_CO2 = text2num(signal.data["co2_scrub"])
+		toggle_gas_scrub(CARBON_DIOXIDE, text2num(signal.data["co2_scrub"]))

```

Comment:
The issue is that a lot of it is embedded in Air Alarm code because of the signal data in its UI and Topic.

I'm planning to make that more procedural, but doing so requires cutting into a lot of bloat code and system creation and testing. It would also hold up this PR for even longer.